### PR TITLE
Add global distribution config transformation utility

### PR DESCRIPTION
**Pull request by vekterli**
@geirst @toregge please review

### CODE REVIEW

**Review by toregge — APPROVED** [review]
LGTM

**Comment by bratseth** [comment]
> Less Gallic influence on naming

Lolz

File: storage/src/vespa/storage/common/global_bucket_space_distribution_converter.h [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+
+#pragma once
+
+#include <vespa/vdslib/distribution/distribution.h>
+#include <vespa/config-stor-distribution.h>
+#include <memory>
+
+namespace storage {
+
+struct GlobalBucketSpaceDistributionConverter {
```
Comment (geirst) [inline storage/src/vespa/storage/common/global_bucket_space_distribution_converter.h]: Consider adding class comment

**Review by geirst — APPROVED** [review]
👍 Consider my comment for next PR

**Pull request merged**